packages/camera/camera/example/ios/RunnerTests/CameraMethodChannelTests.m

```diff
@@ -0,0 +1,101 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import camera;
+@import XCTest;
+@import AVFoundation;
+#import <OCMock/OCMock.h>
+
+@interface FLTThreadSafeFlutterResult ()
+@property(readonly, nonatomic) FlutterResult flutterResult;
+@end
+
+@interface MockFLTThreadSafeFlutterResult : FLTThreadSafeFlutterResult
```

Comment:
Needs a comment explaining its role.

Reply:
Can you have a look at my added comment?

Reply:
Looks good, thanks.